doc/examples/workflow_creation.py

```diff
@@ -96,7 +96,15 @@ def run(self, input_files, text_to_append='dipy', out_dir='',
 executable file located in ``bin``.
 """
 
+"""
+
+These lines will import the required files and classes into the python file. The
+first line imports the run_flow method from the flow_runner class and the second
+line imports the AppendTextFlow method from the newly created workflow class.
+"""
+
 from dipy.workflows.flow_runner import run_flow
+from dipy.workflows.AppendTextFlow import AppendTextFlow
```

Comment:
Can you precise above that we create a file named `my_workflow.py` in `dipy/workflows` folder ?
After that, Can you change `from dipy.workflows.AppendTextFlow import AppendTextFlow` to `from dipy.workflows.my_workflow import AppendTextFlow` ?

Reply:
Hey Serge, thanks for going through the changes.

Made the change to 

from dipy.workflows.my_workflow import AppendTextFlow